frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/ExpressionEditorTextfield.tsx

```diff
@@ -216,10 +305,20 @@ class ExpressionEditorTextfield extends React.Component<
     }
   }
 
+  componentWillUnmount() {
+    this.textarea()?.removeEventListener("keypress", this.handleKeypress);
+  }
+
   onSuggestionSelected = (index: number) => {
     const { source, suggestions } = this.state;
     const suggestion = suggestions && suggestions[index];
 
+    if ("footer" in suggestion) {
+      // open link in new window
+      window.open(suggestion.href, "tab");
```

Comment:
```suggestion
      window.open(suggestion.href, "_blank");
```

(see other external links in the codebase)